test/distributed_exec.jl

```diff
@@ -950,7 +949,10 @@ if is_unix() # aka have ssh
     end
 
     print("\n\nTesting SSHManager. A minimum of 4GB of RAM is recommended.\n")
-    print("Please ensure sshd is running locally with passwordless login enabled.\n")
+    print("Please ensure: \n")
+    print("1) sshd is running locally with passwordless login enabled.\n")
+    print("2) Env variable USER is defined and is the ssh user.\n")
```

Comment:
I'd either spell out "environment" here or use all caps to make it clear that it corresponds to the Julia `ENV` variable. (Also these can all be `println` rather than `print(..., "\n")`.)